packages/loader/driver-utils/src/error.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IFluidErrorBase, LoggingError } from "@fluidframework/telemetry-utils";
+
+/** Error indicating an API is being used improperly resulting in an invalid operation. */
+export class UsageError extends LoggingError implements IFluidErrorBase {
```

Comment:
@vladsud There is a `UsageError` class in `container-utils` already, but I think the driver can't consume this package. Is this correct?